src/Components/UserProfileCard.js

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
-import { Button, Card, Icon, Image } from 'semantic-ui-react';
+import { Grid, Button, Card, Icon, Image } from 'semantic-ui-react';
 
 const UserProfileCard = () => {
   return (
     <Card centered>
-     
       <Card.Content>
-      <Image
-        size="small"
-        floated="left"
-        src="https://react.semantic-ui.com/images/avatar/large/matthew.png"
-      />
-        <Card.Header>Arturo</Card.Header>
-        <Card.Meta>
-          <span className="position">CEO</span>
-        </Card.Meta>
-        <Card.Description>Arturo is a musician living in CDMX.</Card.Description>
-
-        <Button centered color="blue" type="submit">
-          Message
-        </Button>
+        <Grid>
+          <Grid.Column mobile={16} tablet={6} computer={8}>
+            <Grid.Row>
+              <Image
+                size="small"
+                floated="left"
+                src="https://react.semantic-ui.com/images/avatar/large/matthew.png"
+              />
+            </Grid.Row>
+          </Grid.Column>
+          <Grid.Column mobile={16} tablet={6} computer={6}>
+            <Grid.Row centered>
+              <Card.Header>Arturo</Card.Header>
+              <Card.Meta>
+                <span className="position">CEO</span>
+              </Card.Meta>
+              <Card.Description>
+                <span className="email">Avelo@gmail.com</span>
+              </Card.Description>
+              <Card.Description>
+                <span className="phone">+22999348585</span>
+              </Card.Description>
+            </Grid.Row>
+            <Grid.Row centered>
+              <Button centered color="blue" type="submit">
+                Message
+              </Button>
+            </Grid.Row>
+          </Grid.Column>
+        </Grid>
       </Card.Content>
       <Card.Content extra>
```

Comment:
After you resize the card by making it fluid try to add one or two items to the card.content extra and see how it looks, just to make it look more similar to the reference layout.